docs/docs/troubleshooting.md

```diff
@@ -28,19 +28,16 @@ environment variables `CONTRAST_LOG_LEVEL`, `CONTRAST_LOG_FORMAT` and
   be enabled for logging, which are disabled by default. Subsystems include:
   `snp-issuer`, `kds-getter`, and `snp-validator`. To enable all subsystems, use
   `*` as the value for this environment variable.
-
-Warnings and error messages from subsystems get printed regardless of whether
-the subsystem is listed in the `CONTRAST_LOG_SUBSYSTEMS` environment variable.
-
-#### Coordinator debug logging
+  Warnings and error messages from subsystems get printed regardless of whether
+  the subsystem is listed in the `CONTRAST_LOG_SUBSYSTEMS` environment variable.
 
 To configure debug logging with all subsystems for your Coordinator, add the
 following variables to your container definition.
 
 ```yaml
 spec: # v1.PodSpec
   containers:
-    image: "ghcr.io/edgelesssys/contrast/coordinator:latest"
+    image: "ghcr.io/edgelesssys/contrast/coordinator"
```

Comment:
```suggestion
    image: "ghcr.io/edgelesssys/contrast/coordinator:latest"
```
Currently, container images will only get replaced with the tagged version on release when they have at least a `:latest` tag or a SHA.